### PR TITLE
Fix with-preact to be defined as commonjs

### PR DESCRIPTION
**Pull request by cristianbote**
**What**:

Change the format for `@frontity/with-preact` config file to commonjs since it's not getting transpiled if installed and use from `node_modules`.

**Why**:

It breaks compilation otherwise.

**Tasks**:

- [x] Code
- [x] Update other packages
- [x] Add a changeset (with link to its [Feature Discussion](https://community.frontity.org/c/33) if it exists)

**Unrelated Tasks**

- [ ] TSDocs
- [ ] TypeScript
- [ ] Unit tests
- [ ] End to end tests
- [ ] TypeScript tests
- [ ] Update starter themes
- [ ] Update community discussions

**Additional Comments**

### CODE REVIEW

File: .changeset/shaggy-laws-sniff.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@frontity/with-preact": minor
+---
+
+with-preact is not usually transpiled when is part of the node_modules so the config file should be defined as commonjs
```
Comment (luisherranz) [inline .changeset/shaggy-laws-sniff.md]:
```suggestion
"@frontity/with-preact": patch
---

Use `commonjs` in the `frontity.config.js` file until we add support for modules.
```

**Review by luisherranz — APPROVED** [review]
Hey Cris, I have been talking about this with @nicholasio and I think we should find a way to transpile those config files, shouldn't we?

I'm going to open an issue.

We can merge this PR until we fix that so the package is usable 🙂

**Comment by cristianbote** [comment]
Hey @luisherranz thanks for the review. Yeah, it'll be best to define a rule frontity.config files.

**Comment by luisherranz** [comment]
Issue opened: https://github.com/frontity/frontity/issues/835

**Comment by nicholasio** [comment]
> Hey Cris, I have been talking about this with @nicholasio and I think we should find a way to transpile those config files, shouldn't we?
> 
> I'm going to open an issue.
> 
> We can merge this PR until we fix that so the package is usable 🙂

Definitely think we should fix this. The TS defs are really nice for modifying the webpack config.

**Pull request merged**